### ISSUE TITLE
Ready for CRAN?

### ISSUE THREAD

**Issue by MatthewBJane** [MatthewBJane/theme_park#38]
Hey just wanted to see if we are ready for submitting this to CRAN yet? Is the only thing left including some tests? @christopherkenny mentioned he would add those so if that is all we need then that's awesome! Take your time Chris, I'm just curious if there is anything else I could do to get this CRAN ready

**Comment by christopherkenny** [comment]
Yeah, I think the only thing we really need now is some tests. The existing checks seem to run well and everything is structured fine, so shouldn't be a painful submission.

**Comment by christopherkenny** [comment]
Tests are added. One which runs the theme code and one which actually produces a plot if `vdiffr` is available. They're pretty simple but shouldn't be too restrictive.

**Comment by MatthewBJane** [comment]
Awesome thanks a ton Chris! Okay, @christopherkenny and @lukepilling I just followed some online instructions and now the package is submitted to CRAN!!

**Comment by lukepilling** [comment]
Amazing, well done guys. I've learned a lot about package development - thanks @christopherkenny

**Comment by MatthewBJane** [comment]
So recieved a reply back and it did not pass initial checks but I think it was the URL was wrong (it was matthewbjane.com/theme_park instead of matthewbjane.com/ThemePark). I re-submitted it

**Comment by christopherkenny** [comment]
Oops, forgot to check that. There's a little package[ `urlchecker`](https://github.com/r-lib/urlchecker).

I suspect they may also be unhappy with the version number. We probably should have updated 0.0.0.9000 -> 0.0.1 or 0.1.0. Sorry forgot about checking the description details extra carefully for first submission.

**Comment by erikgahner** [comment]
First, thanks for a great package! A lot of interesting and fun themes.

Is there any update on this? I was looking for the package on CRAN to no avail.

**Comment by christopherkenny** [comment]
Good to go by me. Just fixed a small docs thing with the Elf theme.

**Comment by MatthewBJane** [comment]
Great! I will attempt to submit it today!
